tools/ci_build/github/linux/ort_minimal/build_full_ort_and_create_ort_files.sh

```diff
@@ -62,9 +64,9 @@ python3 /onnxruntime_src/tools/python/create_reduced_build_config.py --format OR
 mkdir /home/onnxruntimedev/.test_data/custom_ops_model
 cp /onnxruntime_src/onnxruntime/test/testdata/custom_op_library/*.onnx /home/onnxruntimedev/.test_data/custom_ops_model/
 python3 /onnxruntime_src/tools/python/convert_onnx_models_to_ort.py \
-    --custom_op_library /build/Debug/libcustom_op_library.so \
+    --custom_op_library ${BUILD_DIR}/Debug/libcustom_op_library.so \
     /home/onnxruntimedev/.test_data/custom_ops_model
 rm -rf /home/onnxruntimedev/.test_data/custom_ops_model
 
 # Clear the build
-rm -rf /build/Debug
+rm -rf ${BUILD_DIR}
```

Comment:
If we are using different folders for build output, do we still need to remove it?

Reply:
I guess not